release_dashboard/lib/logic/engine_or_framework.dart

```diff
@@ -0,0 +1,15 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import '../enums/engine_or_framework.dart';
+
+/// Helper function that returns the string 'engine' or 'framework' based on the [engineOrFramework] enum.
+///
+/// Also supports returning first letter capitalized string.
+String engineOrFrameworkStr(EngineOrFramework engineOrFramework, [bool? capitalized = false]) {
+  if (capitalized == true) {
+    return engineOrFramework == EngineOrFramework.engine ? 'Engine' : 'Framework';
```

Comment:
Apparently, it is not straight-forward to make a logic to capitalize the first letter: https://stackoverflow.com/questions/29628989/how-to-capitalize-the-first-letter-of-a-string-in-dart

Returning the string hard-coded instead.

Reply:
`'${engineOrFramework.toUpperCase()}${this.substring(1)}'`

As a side note, engineOrFramework should really just be named repository.

Reply:
Done, please take another look.